components/breadcrumbs/breadcrumb.js

```diff
@@ -93,12 +88,9 @@ class Breadcrumb extends RtlMixin(LitElement) {
 	}
 
 	render() {
-		const icon = this._compact ? html`<d2l-icon icon="d2l-tier1:chevron-left"></d2l-icon>` : html`<d2l-icon icon="d2l-tier1:chevron-right"></d2l-icon>`;
-		return html`
-			<div class="d2l-breadcrumb-wrapper" ?data-compact=${this._compact}>
-				<a class="d2l-link d2l-link-small" aria-label="${ifDefined(this.ariaLabel)}" href="${this.href}" target="${ifDefined(this.target)}">${this.text}</a>${icon}
-			</div>
-		`;
+		const iconStart = this._compact ? html`<d2l-icon icon="d2l-tier1:chevron-left" class="d2l-breadcrumb-icon-start"></d2l-icon>` : null;
+		const iconEnd = !this._compact ? html`<d2l-icon icon="d2l-tier1:chevron-right"></d2l-icon>` : null;
+		return html`${iconStart}<a class="d2l-link d2l-link-small" aria-label="${ifDefined(this.ariaLabel)}" href="${this.href}" target="${ifDefined(this.target)}">${this.text}</a>${iconEnd}`;
```

Comment:
I was able to remove the `d2l-breadcrumb-wrapper` div and the `data-compact` attribute just by either rendering the icon before or after.